test_runner/fixtures/neon_fixtures.py

```diff
@@ -434,10 +429,14 @@ def __init__(
         self.repo_dir = repo_dir
         self.rust_log_override = rust_log_override
         self.port_distributor = port_distributor
-        self.remote_storage = remote_storage
+
+        # Pageserver remote storage
+        self.pageserver_remote_storage = pageserver_remote_storage
+        # Extensions remote storage
         self.ext_remote_storage: Optional[S3Storage] = None
-        self.remote_storage_client: Optional[S3Client] = None
-        self.remote_storage_users = remote_storage_users
+        # Safekeepers remote storage
+        self.sk_remote_storage: Optional[RemoteStorage] = None
```

Comment:
I don't know if this should be done. Is there an upside? I think part of the type defs come from the ctor, but I don't see any use for them outside of that.

Reply:
Why it shouldn't be done, in my mind:
- you cannot configure a remote storage without having a NeonEnvBuilder because of the gazillion args the `RemoteStorageKind.configure` needs

Reply:
Makes sense!
Let's leave it as is